sub-packages/bionemo-core/src/bionemo/core/utils/subprocess_utils.py

```diff
@@ -0,0 +1,54 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: LicenseRef-Apache2
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import logging
+import subprocess
+from typing import Any, Dict, List
+
+
+logger = logging.getLogger(__name__)
+
+
+def run_subprocess_safely(command: List[str], timeout: int = 2000) -> Dict[str, Any]:
+    """Run a subprocess and raise an error if it fails.
```

Comment:
you don't raise an error here right? you're just returning that exception as a dict?